frontend/javascripts/oxalis/model/helpers/nml_helpers.ts

```diff
@@ -502,8 +502,11 @@ function serializeTreeGroups(treeGroups: Array<TreeGroup>, trees: Array<Tree>):
       serializeTagWithChildren(
         "group",
         {
-          id: treeGroup.groupId,
-          name: treeGroup.name,
+          ...{
+            id: treeGroup.groupId,
+            name: treeGroup.name,
+          },
+          ...(treeGroup.isExpanded === false && { isExpanded: treeGroup.isExpanded }),
```

Comment:
didn't know that `false` was spreadable anyway. I think, it's cleaner with a ternary:

```suggestion
          ...(treeGroup.isExpanded ? {} : { isExpanded: treeGroup.isExpanded }),
```